spec/facility_spec.rb

```diff
@@ -15,12 +19,257 @@
   end
 
   describe '#add service' do
-    it 'can add available services' do
+   it 'can add available services' do
       expect(@facility.services).to eq([])
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
       @facility.add_service('Vehicle Registration')
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#vehicle registration' do
```

Comment:
Overall your tests are looking good, just really be mindful of unnecessary white space at the end of your tests and the end of the file.